src/clients/AcrossConfigStoreClient/AcrossConfigStoreClient.ts

```diff
@@ -166,6 +167,27 @@ export class AcrossConfigStoreClient extends BaseAbstractClient {
     return liteChainIdList.find((update) => update.blockNumber <= blockNumber)?.value ?? [];
   }
 
+  /**
+   * Resolves the lite chain ids that were available to the protocol at a given timestamp.
+   * @param timestamp Timestamp to search for. Defaults to latest time - in seconds.
+   * @returns List of lite chain IDs that were available to the protocol at the given timestamp.
+   * @note This dynamic functionality has been added after the launch of Across.
+   */
+  getliteChainIdIndicesForTimestamp(timestamp: number = Number.MAX_SAFE_INTEGER): number[] {
+    const liteChainIdList = sortEventsDescending(this.liteChainIndicesUpdates);
+    return liteChainIdList.find((update) => update.timestamp <= timestamp)?.value ?? [];
```

Comment:
what does .value resolve to? are you sure that's the right syntax?

Reply:
The variable `liteChainIndicesUpdates is defined as 
```
public liteChainIndicesUpdates: LiteChainsIdListUpdate[] = [];
```

The call to 
```
liteChainIdList.find((update) => update.timestamp <= timestamp)
```
returns
```
LiteChainsIdListUpdate<...> | undefined
```
which is defined as
```
export interface LiteChainsIdListUpdate<ValueStore = number[]> extends GlobalConfigUpdate<ValueStore> {
  timestamp: number;
}
```
and has its value stored in 
```
export interface GlobalConfigUpdate<ValueStore = number> extends SortableEvent {
  value: ValueStore;
}
```
which in this case `ValueStore = number[]`